internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Faust Update - January 4, 2023
```

Comment:
Our naming convention is currently `Sprint ## Update` - is this title an intentional deviation from that?

Reply:
Yes. I'm changing it for 2023 as "sprint" is confusing to some... not to mention we're back to sprint 1 which could get really confusing later this year.